tekton/README.md

```diff
@@ -31,25 +31,55 @@ This cluster
 
 To make a new release:
 
+1. [Create draft release](#create-draft-release) in GitHub with release notes
 1. (Optionally) [Apply the latest versions of the Tasks + Pipelines](#setup)
-2. (If you haven't already)
+1. (If you haven't already)
    [Install `tkn`](https://github.com/tektoncd/cli#installing-tkn)
-3. [Run the Pipeline](#run-the-pipeline)
-4. Create the new tag and release in GitHub
+1. [Run the Pipeline](#run-the-pipeline)
+1. Create the new tag and release in GitHub
    ([see one of way of doing that here](https://github.com/tektoncd/pipeline/issues/530#issuecomment-477409459)).
    _TODO(tektoncd/pipeline#530): Automate as much of this as possible with
    Tekton._
-5. Add an entry to [the README](../README.md) at `HEAD` for docs and examples
+1. Add an entry to [the README](../README.md) at `HEAD` for docs and examples
    for the new release ([README.md#read-the-docs](README.md#read-the-docs)).
-6. Update the new release in GitHub with the same links to the docs and
+1. Update the new release in GitHub with the same links to the docs and
    examples, see
    [v0.1.0](https://github.com/tektoncd/pipeline/releases/tag/v0.1.0) for
    example.
 
+### Create draft release
+
+The Task `create-draft-triggers-release` calculates the list of PRs merged between the previous release and a specified revision. It also builds a list of authors and uses PRs and authors to build a draft new release in GitHub.
+
+Running this Task multiple times will create multiple drafts; old drafts have to be pruned manually when needed.
+
+Once the draft release is created, the release manager needs to edit the draft, arranging PRs in the right category, highlighting important changes.
+
+Parameters:
+
+- `package`
+- `release-tag`
+- `previous-release-tag`
+
+Resources:
+
+- `source`, a git resource that points to the release git revision
+
+This Task expects a secret named `github-token` to exists, with a GitHub token in `GITHUB_TOKEN` with enough privileges to list PRs and create a draft release.
+
+```bash
+tkn task start \
+  -i source=tekton-triggers-git \
+  -p package=tektoncd/triggers \
+  -p release-tag=v0.3.1 \
```

Comment:
Should we change this to v.x.yz. or follow the convention below where it is set via an environment variable? (Mainly to prevent the inevitable scenario when someone (read I) copy pastes the entire thing overwriting/creating additional drafts)

Reply:
Yep, good point 😅